### ISSUE TITLE
Make the module name/path more consistent

### ISSUE THREAD

**Issue by jinq102030** [rapid7/metasploit-framework#8510]
## Steps to reproduce

How'd you do it?

1. ./msfconsole
2. use exploits/linux/http/kloxo_sqli
3. 
```
msf exploit(kloxo_sqli) > show options

Module options (exploit/linux/http/kloxo_sqli):
```

Notice the word "exploits" in the "use ..." command and "exploit" in the module name as the output of "show options"?

Sometimes when I need to open the actual ruby file for the module, it's easy to specify the file path by copying "exploit/linux/http/kloxo_sqli" from "show options" command and appending the .rb, but then you have to remember it's "exploits" not "exploit", which is quite annoying.

**Comment by wvu-r7** [comment]
This is almost a duplicate of #7693 and should be resolved by #7734.

**Comment by wvu-r7** [comment]
Oh, you're asking something else. No, I think that should stay as a reference name. You can use the `edit` command to open the file in your editor.
